src/query/service/src/table_functions/show_sequences/show_sequences_table.rs

```diff
@@ -81,8 +80,6 @@ impl ShowSequences {
     fn schema() -> Arc<TableSchema> {
         TableSchemaRefExt::create(vec![
             TableField::new("name", TableDataType::String),
-            TableField::new("start", TableDataType::Number(NumberDataType::UInt64)),
-            TableField::new("interval", TableDataType::Number(NumberDataType::Int64)),
```

Comment:
Need this schema.

Reply:
why?

Reply:
User need to see the seq start and interval. Snowflake also display these value. If these fields can be delete in meta we need to Cala them used to display.